script/Systems/Universe/Universe.lua

```diff
@@ -114,7 +114,9 @@ function Universe:CreateShip(system, pos, shipObject)
     ship:setName(shipObject.shipName)
 
     -- Insert ship into this star system
-    local spawnPosition = pos or Config.gen.origin
+    local spawnPosition = ship:getPos() -- use semi-randomly generated position from spawnShip()
```

Comment:
I must have been pretty tired when i approved this :D - now that i think of this it´s kinda a bad fix, also generating a ship was working before, better we fix this in hud.lua instead

Reply:
also this change now bugs out the camera in that generation screen